brianlong/www.validators.app
# 186710123 skipped after history on validator details

#### What's this PR do?
- add skipped after chart to validator details page

#### How should this be manually tested?
- run gather and score daemons (make sure you have some historical data in validator_block_histories)
- go to any validator detail page
- make sure there are four charts - the last one should be skipped after chart
- check if it looks correct

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186710123)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
